cairo/tests/ethereum/cancun/vm/instructions/test_system.py

```diff
@@ -55,3 +59,27 @@ def test_return(self, cairo_run, evm: Evm):
 
         return_(evm)
         assert evm == cairo_result
+
+    @given(
+        evm=EvmBuilder()
+        .with_env()
+        .with_message()
+        .with_stack()
+        .with_gas_left()
+        .with_running()
+        .with_accessed_addresses()
+        .with_accessed_storage_keys()
+        .with_accounts_to_delete()
+        .with_touched_accounts()
+        .build()
```

Comment:
are we sure all branches are triggered?
No need for a composite strategy to ensure all address are from the state?